app/components/subject-viewer.cjsx

```diff
@@ -260,6 +267,9 @@ module.exports = createReactClass
       mods[type] = not mods[type]
     @setState modification: mods
 
+  toggleInvert: (invert) ->
+    @setState invert: !invert
```

Comment:
```suggestion
    @setState (prevState) -> { invert: !prevState.invert }
```
You can modify previous state inside setState by using a callback, rather than passing current state in.